### ISSUE TITLE
XML formatting of empty elements forces inclusion of enwline and tabs in element value

### ISSUE THREAD

**Issue by bmrussell** [rangav/thunder-client-support#600]
**Describe the bug**
The `format` button seems to be inserting newlines into empty elements and aligning the open/close tag with tabs. This can cause issues as the indentation tabs can be picked up as the value of the element

**To Reproduce**
Add an xml request containing
```
					<tag></tag>
```
**Observed behavior**
```
					<tag>
                                        </tag>
```
**Expected behavior**
```
					<tag></tag>
```

**Platform:** 
 - OS: Windows
 - vscode version: 1.67.2
 - extension version: v1.16.4

**Workaround:**
Format XML with vscode document

**Comment by rangav** [comment]
HI @bmrussell thanks for reporting the issue, will look into it.

**Comment by rangav** [comment]
This bug is is now fixed and published to marketplace, please update to `v1.18.3`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v1.18.3

Please let me know your feedback.
